website/docs/docs/get-started/getting-started/getting-set-up.md

```diff
@@ -2,10 +2,10 @@
 title: Set up dbt Cloud
 id: set-up-dbt-cloud
```

Comment:
Should this match the page name? I'm not sure what this does for linking?

Reply:
not always. here's one example: https://github.com/dbt-labs/docs.getdbt.com/blob/next/website/docs/docs/deploy/deployment-overview.md